demos/using-dev-build/test.ts

```diff
@@ -57,19 +57,16 @@ function parseCanisterIDs(): CanisterIDs {
   }
 }
 
-function parseReplicaHost(): string {
-  const DFX_JSON_PATH = `${__dirname}/dfx.json`;
-  try {
-    const dfx_json = JSON.parse(fs.readFileSync(DFX_JSON_PATH, "utf8"));
-    let replicaHost = dfx_json.networks.local.bind;
-    if (!replicaHost.startsWith("http://")) {
-      replicaHost = `http://${replicaHost}`;
-    }
-    return replicaHost;
-  } catch (e) {
-    console.log(`Could not read replica host from ${DFX_JSON_PATH}`);
-    throw e;
-  }
+async function getReplicaHost(): Promise<string> {
+  let proc = spawn("dfx", ["info", "webserver-port"]);
+  let result = "";
```

Comment:
what is `result` for?

Reply:
It was used as an accumulator but no longer necessary.